.github/workflows/kube-prow-bot.yaml

```diff
@@ -0,0 +1,166 @@
+name: Kube Prow Bot
+on:
+  issue_comment:
+    types:
+      - created
+
+  pull_request_review_comment:
+    types:
+      - created
+
+  issues:
+    types:
+      - opened
+
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
+  pull_request_target:
+    types:
+      - opened
+
+env:
+  # This commands is for anyone who can use it
+  COMMON_PLUGINS: |-
+    assign
+    unassign
+    kind
+    remove-kind
+    cc
+    uncc
+  # This commands is for author of issue or PR
+  AUTHOR_PLUGINS: |-
+    retest
+  # This commands is for organization member or repository member
+  MEMBERS_PLUGINS: |-
+    good-first-issue
+    help-wanted
+    close
+    reopen
+    retest
+  # This commands is for in the REVIEWERS environment variable
+  REVIEWERS_PLUGINS: |-
+    area
+    remove-area
+    lgtm
+    hold
+    unhold
+  # This commands is for in the APPROVERS environment variable
+  APPROVERS_PLUGINS: |-
+    merge
+    approve
+    rebase
+  # This commands is for in the MAINTAINERS environment variable
+  MAINTAINERS_PLUGINS: |-
+    milestone
+    remove-milestone
+    priority
+    remove-priority
+
+  # REVIEWERS: |-
+  #   Xunzhuo
+  # APPROVERS: |-
+  #   Xunzhuo
+  MAINTAINERS: |-
+    AliceProxy
```

Comment:
can we reuse OWNER file?

Reply:
Yes. It sounds good and we can follow: https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md.  But I plan to work on next release.

Reply:
This is already be done in prowox.